scripts/tests/test_install_pi_transaction.py

```
@@ -93,6 +93,24 @@ def test_retired_pi_models_are_pruned_without_claiming_custom_entries(self):
             full=subprocess.run(['bash','scripts/verify-pi-install.sh','--check-only'],cwd=ROOT,env=env,stdout=subprocess.PIPE,stderr=subprocess.PIPE,text=True)
             self.assertNotEqual(full.returncode,0);self.assertIn('enabledModels still contains retired GPT-5.4 Pi routes',full.stdout)
 
+    def test_opencode_grok_4_5_survives_retired_grok_normalization(self):
+        with tempfile.TemporaryDirectory() as d:
+            home=Path(d);agent=home/'.pi/agent';(agent/'agents').mkdir(parents=True)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)